src/model/entities/rundown.ts

```diff
@@ -50,10 +51,13 @@ export class Rundown extends BasicRundown {
 
   private persistentState?: RundownPersistentState
 
+  private readonly showStyleVariantId: string
```

Comment:
I'm not a fan of `Rundown` being the one that knows which `ShowStyleVariant` has been selected.
The `Rundown` has the responsibility of keeping track of its Segments and Parts and ensuring they are in the right state.

Which ShowStyleVariant that has been selected is out of the scope of what `Rundown` should know.

But it's possible to have different variants selected for different Rundowns, so we would need some way to map them, which might be out-of-scope for this task.

Leave it as is for now :) Anders and I need to have a talk about this.